examples/realm/ts/tsconfig.json

```diff
@@ -13,6 +13,6 @@
         "forceConsistentCasingInFileNames": true
     },
     "files": [
-        "index.ts"
+      "ts/index.ts"
```

Comment:
Why `ts` here? It's already in the `ts` folder.

Reply:
fixed! Good spot - refactoring tools eh ;)